src/main/java/uk/gov/ida/eidas/utils/FileReader.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.ida.eidas.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class FileReader {
+    public static String readFileContent(File file) throws IOException {
+        return readFileContent(file.toPath());
+    }
+
+    public static String readFileContent(String filePath) throws IOException {
+        return readFileContent(Paths.get(filePath));
+    }
+
+    public static String readFileContent(Path path) throws IOException {
+        return new String(Files.readAllBytes(path), StandardCharsets.UTF_8);
```

Comment:
I don't know how good the IOException error messages are but in general it would be nice to wrap this exception with a custom one with a custom message.

Reply:
I think IOException has pretty readable messages and it takes care of all the reasonings (if a file doesn't exist, not writeable, etc etc) for us.